### ISSUE TITLE
Any plans for merging the icc color profile support recently added to sway?

### ISSUE THREAD

**Issue by kockahonza** [WillPower3309/swayfx#348]
Pretty much the title. I am talking about [this pull request](https://github.com/swaywm/sway/pull/7681) that has added the `color_profile` option for the `output` command. I ask as it is pretty major feature for me given how oversaturated colors are without it on my screen.

**Comment by WillPower3309** [comment]
Happy to backport this change to swayfx early, not on my todo list, but open to PRs

**Comment by WillPower3309** [comment]
otherwise we can get it for free next sway release

**Comment by kockahonza** [comment]
Oh I see, so swayfx always fully catches up to any changes in sway with their releases (with major changes being backported sooner)? I was wondering about how this was managed.

When it comes to adding this earlier it's not that essential for me. I have gone back to sway for now but I am perfectly happy to wait for a while and I don't really have the time at the moment to work on this and I expect the same applies to others. I'm of course happy to test though :) and I am looking forward to switching when it gets there.

**Comment by AleMax** [comment]
Hi, sorry if I am wrong here, but considering this needs the vulkan renderer of wlr_renderer, are we sure that we can just get this for free with the next sway release?

**Comment by WillPower3309** [comment]
> Hi, sorry if I am wrong here, but considering this needs the vulkan renderer of wlr_renderer, are we sure that we can just get this for free with the next sway release?

You could be right, I haven't dove into the implementation yet. If it does require Vulkan than yes we won't get it yet :/

**Comment by markstos** [comment]
Seems like this will be resolved by #360, rebasing off Sway 1.10.

**Comment by WillPower3309** [comment]
> Seems like this will be resolved by #360, rebasing off Sway 1.10. 

Yup, but if it needs the Vulkan renderer this wouldn't be the case. Still need to dive into the implementation to determine that

**Comment by rhoot** [comment]
Sadly it does require the wlroots vulkan renderer. The upstream wlroots merge request does most of the heavy lifting: https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/4280

**Comment by WillPower3309** [comment]
> Any updates on this? I do know that swayfx has been rebased off sway 1.10 (or better to say, 1.11 now) for some time. However, I can't still seem to make the icc profiles work on swayfx. On sway as long as the vulkan renderer flag is set everything is in order. Doesn't work on swayfx though. I'm going to guess this is due to the lack of vulkan, correct?
> 
> Also it seems like this has been in the plan for some time now: [#11](https://github.com/WillPower3309/swayfx/issues/11)

Yup correct, vulkan is on the roadmap though. I happen dived into the icc stuff but it may be possible with our gles3 shaders as well
